why_choose_vinta.md

```diff
@@ -0,0 +1,13 @@
+Bill Gates once mentioned that: **“Your most unhappy customers are your greatest source of learning.”** Sometimes that points you to focusing more on the client’s needs, other times it warns you that you might be looking at the wrong customer. Here we describe the things Vinta looks for as the right customer type. Clients like these are the ones Vinta can deliver most value to. These aspects came from our experience in aligning business value with Vinta team’s values.
```

Comment:
I think it's "Vinta's team values" instead of "Vinta team's values". But worth confirming.

Reply:
changed to Vinta's team's values, what do you think?